atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/AtlasDbEteServer.java

```diff
@@ -87,6 +89,8 @@ public static void main(String[] args) throws Exception {
     public void initialize(Bootstrap<AtlasDbEteConfiguration> bootstrap) {
         bootstrap.setMetricRegistry(SharedMetricRegistries.getOrCreate("AtlasDbTest"));
         enableEnvironmentVariablesInConfig(bootstrap);
+        bootstrap.setObjectMapper(
+                ObjectMappers.newServerObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver()));
         bootstrap.getObjectMapper().registerModule(new Jdk8Module());
```

Comment:
nit: I think this gets auto-added by ObjectMappers.newServerObjectMapper (in fact TIL that we configureAbsentsAsNulls). Regardless we probably don't want to register this twice.